cmake/python.cmake

```diff
@@ -31,7 +31,12 @@ if(${SWIG_VERSION} VERSION_GREATER_EQUAL 4)
   list(APPEND CMAKE_SWIG_FLAGS "-doxygen")
 endif()
 
-if(UNIX AND NOT APPLE)
+# On OpenBSD, int64_t is long long int
+if(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+  list(APPEND CMAKE_SWIG_FLAGS "-DSMALL_LONG")
```

Comment:
not sure you need this. `SMALL_LONG` is not part of the SWIG implem nor or-tools to have `long long int` simply **NOT** define `-DSWIGWORDSIZE64` (like you did below adding the "AND NOT OpenBSD")
ref: https://github.com/swig/swig/blob/v4.2.1/Lib/stdint.i

note: The only reference to `SMALL_LONG` was in https://github.com/swig/swig/issues/1557


side note: for Java and C# with fix the typemap here
https://github.com/google/or-tools/blob/main/ortools/base/base.i

Reply:
I will kick off a build with the "SMALL_LONG" flag removed. Will report back